docs/querybuilder.rst

```diff
@@ -174,11 +174,27 @@ Now we have everything we need to write and execute our first query!
     }
     run();
 
+We can also run queries built with the querybuilder in a transaction.
+
+.. code-block:: typescript
+
+    // script.ts
+    import {createClient} from "edgedb";
+    import e from "./dbschema/edgeql-js";
+
+    const client = createClient();
+
+    const query = e.select(e.datetime_current());
+    client.transaction(async tx => {
+      const result = await query.run(tx);
+      console.log(result);
+    });
+
```

Comment:
Maybe this example should be in a note block at the end of the section, so it doesn't break up the flow of the 'run a first query' mini tutorial, and make it stand out a bit. Also, probably can skip the imports and `createClient` lines.

Reply:
By "end of the section," do you mean at the end of the quickstart section? Would you maybe put it in the cheatsheet section as another example, or is that too buried?

Reply:
Yes, end of the quickstart section.

Reply:
Thank you. Got it moved.